controllers/messages/MessageController.ts

```diff
@@ -31,7 +31,7 @@ export default class MessageController implements IMessageController {
     const router: Router = Router();
     router.get(
       '/:userId/messages',
-      isAuthenticated,
+      // isAuthenticated,
```

Comment:
```suggestion
      isAuthenticated,
```

Don't we need this to be authenticated?